managed/SavedSearch_listings.mgd.php

```diff
@@ -0,0 +1,133 @@
+            'title',
+            'subtype:label',
+            'created_date',
+            'modified_date',
+          ],
+          'orderBy' => [],
+          'where' => [],
+          'groupBy' => [],
+          'join' => [],
+          'having' => [],
+        ],
+        'expires_date' => NULL,
+        'description' => NULL,
+        'mapping_id' => NULL,
+      ],
+    ],
+  ];
+  $searches[] = [
+    'name' => 'SavedSearch_listing_display:' . $type['name'],
+    'entity' => 'SearchDisplay',
+    'cleanup' => 'always',
+    'update' => 'unmodified',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'name' => 'ECK_Listing_Display' . $type['name'],
+        'label' => E::ts('ECK Listing Display: %1', [1 => $type['label']]),
+        'saved_search_id.name' => 'ECK_Listing_' . $type['name'],
+        'type' => 'table',
+        'settings' => [
+          'actions' => TRUE,
+          'limit' => 50,
+          'classes' => [
+            'table',
+            'table-striped',
+          ],
+          'pager' => [
+            'show_count' => TRUE,
+            'expose_limit' => TRUE,
+          ],
+          'placeholder' => 5,
+          'sort' => [],
+          'columns' => [
+            [
+              'type' => 'field',
+              'key' => 'title',
+              'dataType' => 'String',
+              'label' => E::ts('Title'),
+              'sortable' => TRUE,
+              'editable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'subtype:label',
+              'dataType' => 'String',
+              'label' => E::ts('Type'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'created_date',
+              'dataType' => 'Timestamp',
+              'label' => E::ts('Created'),
+              'sortable' => TRUE,
+              'rewrite' => E::ts('[created_date] by [created_id.display_name]'),
+            ],
+            [
+              'type' => 'field',
+              'key' => 'modified_date',
+              'dataType' => 'Timestamp',
+              'label' => E::ts('Modified'),
+              'sortable' => TRUE,
+              'rewrite' => E::ts('[modified_date] by [modified_id.display_name]'),
+            ],
+            [
+              'type' => 'buttons',
+              'alignment' => 'text-right',
+              'size' => 'btn-xs',
+              'links' => [
+                [
+                  'entity' => $type['entity_name'],
+                  'action' => 'view',
+                  'join' => '',
+                  'target' => '_blank',
+                  'icon' => 'fa-external-link',
+                  'text' => E::ts('View'),
+                  'style' => 'default',
+                  'path' => '',
+                  'condition' => [],
+                ],
+                [
+                  'entity' => $type['entity_name'],
+                  'action' => 'update',
```

Comment:
The *Edit* link points directly to the afform instead of the tabset with the *Edit* tab open. I'm not sure where the SearchKit display takes the path for that action from

Reply:
See above comment. If we get it working as a popup then the direct link is best.